src/components/snackbar/IconSnackbar.tsx

```diff
@@ -0,0 +1,22 @@
+import { BoxProps } from "@mui/material";
```

Comment:
can you try to name files with hyphens and lowercase?

eg. icon-snackbar.tsx

small thing but it just makes it more readable with big component folders, ty:)